src/Tizen.Applications.Common/Tizen.Applications.RPCPort/Port.cs

```diff
@@ -42,5 +42,56 @@ public enum Type
 
         internal IntPtr Handle { get; set; }
         internal Port() { }
+
+        /// <summary>
+        /// Shares private files with other applications.
+        /// </summary>
+        /// <remarks>
+        /// If all added paths are under the caller application's data path which can be obtained, those will be shared to the target application.
+        /// Platform will grant a temporary permission to the target application for those files and revoke it when the target application is terminated or UnshareFile() is called.
+        /// Paths should be regular files. The target application can just read them.
+        /// Note that the target application doesn't have read permission of the directory that is obtained by caller's data path. You should open the file path with read only mode directly.
+        /// </remarks>
+        /// <param name="paths">The file paths to be shared.</param>
+        /// <exception cref="InvalidIOException">Thrown when an internal IO error occurrs.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public void ShareFile(string[] paths)
```

Comment:
✔️ DO prefer using collections over arrays in public APIs. The Collections section provides details about how to choose between collections and arrays.
```suggestion
        public void ShareFile(IEnumerable<string> paths)
```